src/abi/generation.rs

```diff
@@ -0,0 +1,149 @@
+use crate::cargo::manifest::CargoManifestPath;
+use quote::{format_ident, quote};
+use std::path::{Path, PathBuf};
+use std::{collections::HashSet, fs};
+use toml::value;
+
+pub(crate) fn generate_toml(manifest_path: &CargoManifestPath) -> anyhow::Result<String> {
+    let original_cargo_toml = fs::read_to_string(&manifest_path.path)?;
+    let original_cargo_toml: toml::value::Table = toml::from_str(&original_cargo_toml)?;
+
+    let mut near_sdk = original_cargo_toml
+        .get("dependencies")
+        .ok_or_else(|| anyhow::anyhow!("[dependencies] section not found"))?
+        .get("near-sdk")
+        .ok_or_else(|| anyhow::anyhow!("near-sdk dependency not found"))?
+        .as_table()
+        .ok_or_else(|| anyhow::anyhow!("near-sdk dependency should be a table"))?
```

Comment:
My assumption is that `abi` is going to stay a feature in `near-sdk` (that might become enabled by default in the future).

@austinabell do you agree with my understanding?

Reply:
I agree, but also `abi` might be a default feature in the future, so this wouldn't have to be specifically defined

Reply:
yeah, so when it's not specifically defined, then the entry won't be a table. meaning this would force users to do this:

```toml
near-sdk = { version = "5.0.0" }
```